sql-cli/tests/test_project.py

```diff
@@ -16,6 +16,8 @@
     Path("config/default/configuration.yml"),
     Path("config/dev"),
     Path("config/dev/configuration.yml"),
+    Path("config/global"),
```

Comment:
@pankajkoti, what do you think if we wrote a test that tried to reproduce @mikeshwe 's initial problem - and confirmed that the current solution works?

Reply:
Added test `test_generate_custom_airflow_config` in `test___main__.py`